src/System.Reflection.Metadata/tests/System.Reflection.Metadata.Tests.csproj

```diff
@@ -5,8 +5,6 @@
     <ProjectGuid>{7061832A-E8CF-4AB6-A8DC-44D2F5A43A13}</ProjectGuid>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <ExternallyShipping>false</ExternallyShipping>
-    <AssemblyName>System.Reflection.Metadata.Tests</AssemblyName>
```

Comment:
I mentioned this in person, but we need to keep the `RootNamespace` here because it controls the names of the embededded resources that the test code uses. Everything else can go, though.